import "MsgHead.proto";    
message MsgLG2GateLoginACK
{	
	enum enResult
	{
		LG_OK				 = 0;
		LG_ERROR			 = 1;
	}

	required uint64			id = 1;
	required uint64         account = 2; 
	required enResult		result = 3;
}




